bin/wp-env-config.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+## Check if user already exists
+npm run wp-env run tests-cli wp user get customer 2> /dev/null
+
+## if 0 is the exit code then we can leave otherwise we'll try creating the user
+if [ $? -eq 0 ]
+then
+	EXIT_CODE=0
+else
+  npm run wp-env run tests-cli wp user create customer customer@woocommercecoree2etestsuite.com --user_pass=password --role=customer
+  EXIT_CODE=$?
+fi
```

Comment:
This captures exit codes from wp commands so we don't create a fail condition in travis (unless there definitely is something that goes wrong).